### PR TITLE
[bitnami/cassandra] add env for setting auto_snapshot_ttl

### PR DESCRIPTION
**Pull request by nevlkv**
PR on[ issue](https://github.com/bitnami/containers/issues/19750)

### CODE REVIEW

**Comment by nevlkv** [comment]
@mdhont, need review

**Comment by mdhont** [comment]
I'm running some final tests. I'll notify you soon.

File: bitnami/cassandra/4.1/debian-11/rootfs/opt/bitnami/scripts/libcassandra.sh [hunk 1]
```diff
@@ -233,6 +233,10 @@ cassandra_validate() {
         check_resolved_hostname "$seed"
     done
 
+    if [[ ! ${CASSANDRA_AUTO_SNAPSHOT_TTL} =~ ([1-9]{1}[0-9]{0,}[d|h|m]) ]]; then
+       print_validation_error "CASSANDRA_AUTO_SNAPSHOT_TTL accepted units: d (days), h (hours) or m (minutes)"
```
Comment (mdhont) [inline bitnami/cassandra/4.1/debian-11/rootfs/opt/bitnami/scripts/libcassandra.sh:237]: Could you fix the indentation?

```suggestion
        print_validation_error "CASSANDRA_AUTO_SNAPSHOT_TTL accepted units: d (days), h (hours) or m (minutes)"
```

**Review by mdhont — CHANGES_REQUESTED** [review]

**Review by mdhont — APPROVED** [review]
Lgtm! Thanks for contributing.

**Pull request merged**